src/main/resources/db/migration/postgresql/V1_39__3803_ADD_ACTION_SUSPENSION_TABLE.sql

```diff
@@ -0,0 +1,16 @@
+SET search_path TO casework;
+
+CREATE TABLE action_data_suspensions (
+    uuid                        UUID PRIMARY KEY,
+    action_uuid                 UUID NOT NULL, -- knitted to info.schema case_type_action table entry uuid.
```

Comment:
Is there a potential for separate piece here to move case_type_action out of info maybe?

Reply:
I think that possible however is probably dependent on the future architecture of DECS conversation. I'm not sure I would want to do any of it right now.